README.md

```diff
@@ -1,2 +1,9 @@
-# docs-cn
+# TiDB 文档 
+
```

Comment:
I'm not sure about this. I keep this to make someone who does not speak Chinese could understand what's this repo used for.